internal/util/k8s.go

```diff
@@ -63,8 +63,19 @@ func GetKubeNamespace() (string, error) {
 	return string(namespace), nil
 }
 
-func GetChaincodePodObject(imageData ImageJson, namespace, peerID string, chaincodeData ChaincodeJson) *apiv1.Pod {
-	chaincodeImage := imageData.Name + "@" + imageData.Digest
+func GetChaincodePodObject(devMode bool, imageData ImageJson, namespace, peerID string, chaincodeData ChaincodeJson) *apiv1.Pod {
+	var chaincodeImage string
+	var imagePullPolicy apiv1.PullPolicy
+	if devMode {
+		// Ignore the image digest in dev mode to allow chaincode which has not been pushed to
+		// a registry to start
+		// TODO log a suitable health warning!
+		chaincodeImage = imageData.Name + ":latest"
```

Comment:
Please remove the :latest attribute in dev mode.  This is an incorrect convention and has impacts on how Kube loads the image from the local system.

Reply:
I thought Kubernetes would just default to `latest` anyway, but just wanted to make it explicit.

> If you don't specify a tag, Kubernetes assumes you mean the tag `latest`.

Reply:
Kube image labels and pull policy are _subtly intertwined_.   "no tag" does not have the same behavior as "latest."  This is a common assumption but it is incorrect.  

See the [kube docs](https://kubernetes.io/docs/concepts/containers/images/#imagepullpolicy-defaulting) for the gory details.

Reply:
Thanks. This seems like a good explanation of how those gory details relate to loading images in kind...

https://kind.sigs.k8s.io/docs/user/quick-start/#loading-an-image-into-your-cluster